File: front/src/app/shared/models/ClienteDTO.ts
export class ClienteDTO {
    id: number = 0;
    nome: string = '';
    telefone: string = '';
    bairro: string = '';
    rua: string = '';
    numero : string = '';
    bloco: string = '';
    apartamento: string = '';
    complemento: string = '';
    enderecoDescricao: string = '';
}

export class BairroDTO {
    id: number = 0;
    nome: string = '';
    taxaEntrega: number = 0;
}

export class RuaDTO {
    id: number = 0;
    nome: string = '';
}

File: back/src/main/java/br/com/nostramassa/gestao/dtos/pedido/ClienteDTO.java
package br.com.nostramassa.gestao.dtos.pedido;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ClienteDTO {

	private Long id;
	private String nome;
	private String telefone;
	private String bairro;
	private String rua;
	private String numero;
	private String bloco;
	private String apartamento;
	private String complemento;
	private String enderecoDescricao;
	private Double taxaEntrega = 0d;
}
